packages/ui/src/app/pages/Election/components/ElectionProgressBar.tsx

```diff
@@ -27,11 +27,12 @@ interface ElectionProgressBarProps extends StatisticItemProps {
   electionStage: string
 }
 
+const GENESIS_BLOCK_TIMESTAMP = 1670693046000
```

Comment:
It would be nice to be able to convert block/dates without fetching anything from the network, but Pioneer should avoid hardcoding these kind of data because it should potentially work on other networks too. Also basing the future block date based on a date which is that old is that old makes it less accurate.